public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js

```diff
@@ -166,19 +166,21 @@ class DefineMonitor extends Component {
 
   async onRunQuery() {
     const { httpClient, values } = this.props;
+    // console.log("http client: " + httpClient);
```

Comment:
small: remove the unnecessary console log statements here (commented out and non-commented out)